chain/network/src/peer/peer_actor.rs

```diff
@@ -879,13 +888,16 @@ impl PeerActor {
             PeerMessage::RequestUpdateNonce(edge_info) => {
                 ctx.spawn(
                     wrap_future(
-                        self.network_state.peer_manager_addr.send(
-                            PeerToManagerMsg::RequestUpdateNonce(
-                                self.other_peer_id().unwrap().clone(),
-                                edge_info,
+                        {
+                            self.network_state.peer_manager_addr.send(
+                                PeerToManagerMsg::RequestUpdateNonce(
+                                    self.other_peer_id().unwrap().clone(),
+                                    edge_info,
+                                )
+                                .with_span_context(),
                             )
-                            .with_span_context(),
-                        ),
+                        }
```

Comment:
this extra block is not necessary, is it?

Reply:
Removed